src/main/java/games/strategy/triplea/delegate/NonFightingBattle.java

```diff
@@ -161,6 +161,9 @@ public void unitsLostInPrecedingBattle(final IBattle battle, final Collection<Un
     }
   }
 
+  /*
+  * Add dependent units - suppress javadoc error
+  */
```

Comment:
Does this actually suppress the Checkstyle warning?  It looks like a standard non-Javadoc block comment.

Regardless, we probably shouldn't be adding dummy Javadocs just to suppress warnings.  Checkstyle offers explicit mechanisms for silencing individual warnings if we determine it's necessary or desirable to do so.  That makes it easier to go back later and review the decision to suppress those warnings.